tests/org.eclipse.n4js.ide.tests/src/org/eclipse/n4js/ide/tests/contentassist/Issue1756Test.xtend

```diff
@@ -0,0 +1,90 @@
+/** 
+ * Copyright (c) 2020 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * Contributors:
+ * NumberFour AG - Initial API and implementation
+ */
+package org.eclipse.n4js.ide.tests.contentassist
+
+import org.eclipse.n4js.ide.tests.server.AbstractCompletionTest
+import org.junit.Test
+
+/** 
+ */
+class Issue1756Test extends AbstractCompletionTest {
+	
+	@Test
+	def void test01() {
+		testAtCursor('''
+			let value;
+			
+			export function foo(): any {
+			    return (arg) => {
+			        let x = async () => {
+			            value = new C((p: Object) => {
+			            	someFun<|>
+			            });
+			            return null;
+			        };
+				}
```

Comment:
The test code mixes tabs and spaces for indentation (also in other test methods). Please change to tabs.